graphql-schemas/schema.yml

```diff
@@ -2386,6 +2386,11 @@ confs:
   fields:
   - { name: slack, type: boolean }
 
+- name: IntegrationSpecShardSpecOverride_v1
```

Comment:
from a naming perspective i would love to see AWSAccount somwhere in the name of this type, since it is an overwrite specific for per-aws-account sharding

Reply:
wdyt making this an interface? the sharding strategy for an integration is pluggable

Reply:
Good point, I'll take another look.

Reply:
by having an awsAccount mentioned here as mandatory. there can't be any other shard overwrites except for aws shards

Reply:
At this point, we have no other sharding strategies. To be frank, adding interfaces feels like overengineering to me at this point. Shouldn't it be easy to add later on?

Reply:
Consistency wise, this also screams interface to me...

Reply:
Hm, okay. I'll take a look into rewriting it to use interfaces

Reply:
I'll let @geoberle chime in. I understood `shardingStrategy` would be the deciding factor here, not `accountType`